Backend/app/rider-platform/rider-app/Main/src/Tools/Error.hs

```diff
@@ -127,6 +127,32 @@ instance IsHTTPError PersonStatsError where
 
 instance IsAPIError PersonStatsError
 
+data MediaFileError
+  = FileSizeExceededError Text
+  | FileDoNotExist Text
+  | FileFormatNotSupported Text
+  deriving (Eq, Show, IsBecknAPIError)
+
+instanceExceptionWithParent 'HTTPException ''MediaFileError
+
+instance IsHTTPError MediaFileError where
+  toErrorCode = \case
+    FileSizeExceededError _ -> "FILE_SIZE_EXCEEDED"
+    FileDoNotExist _ -> "FILE_DO_NOT_EXIST"
+    FileFormatNotSupported _ -> "FILE_FORMAT_NOT_SUPPORTED"
+  toHttpCode = \case
+    FileSizeExceededError _ -> E400
+    FileDoNotExist _ -> E400
+    FileFormatNotSupported _ -> E400
+
```

Comment:
can we throw 413 error code for file sized exceeded. Please refer to the HTTP error codes, rather than putting 400 everytime. As 400 corresponds to bad request